app/views/questions/_form.html.erb

```diff
@@ -15,7 +15,7 @@
   </div>
   <div class="modal-footer">
     <% if @question.nil? %>
-      <button type="button" class="btn btn-primary" data-dismiss="modal">See Orgs</button>
+      <button type="button" class="btn btn-finish-quiz" data-dismiss="modal">See Orgs</button>
```

Comment:
We should ask the client about the copy for the image hero and buttons in general.